docs/ios/ios-tvos.md

```diff
@@ -1,13 +1,16 @@
 ## tvOS support
 
-- Appium 1.13.0+ has tvOS support via XCUITest driver
-    - It does not work on Apple TV 4K since [ios-deploy](https://github.com/ios-control/ios-deploy) does not work on wireless devices
+tvOS is supported since XCUITest driver bundled with Appium 1.13.0+
 
 <img src="https://user-images.githubusercontent.com/5511591/55161297-876e0200-51a8-11e9-8313-8d9f15a0db9d.gif" width=50%>
 
+> **Warning**
+> Apple TV 4K is not supported because [appium-ios-device](https://github.com/appium/appium-ios-device) library,
```

Comment:
We can update this warning after https://github.com/appium/appium/issues/17506, but maybe tvOS can work with `ios-deploy` installation strategy. (at least newer ios-deploy was able to install apps to network devices. I only tested my local iPhone tho). So... at least xcodebuild for wda + ios-deploy could help.
(Not sure recent apple tv device models, but maybe the initial 4k device model worked before in my testing.)

Reply:
We do not use ios-deploy anymore (unless explicitly asked to in capabilities). Although even if app installation is going to work the whole flow would still be broken. The reason for it is that appium-ios-device does not support local port forwarding over WiFi, so we won't be able to connect to WDA server running on the device

Reply:
ah, actually I often use `webDriverAgentUrl` and iproxy as the local port forwarding to the WDA itself to avoid xcodebuild flakiness. Btw, usbmuxd communication itself seems working since some commands such as `mobile: killApp` and file push/pull worked for Wi-Fi devices on my local iPhone/iPad right now. (haven't tested on 4k tv)